sys/freebsd/kqueue.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <sys/types.h>
+include <sys/event.h>
+
+kqueue() fd
+kevent(fd fd, changelist ptr[in, array[kevent]], nchanges len[changelist], eventlist ptr[in, array[kevent]], nevents len[eventlist], timeout ptr[in, timespec])
```

Comment:
Better to define a new resource for kqueues and use that instead: `resource kqueue[fd]`. See the section on resources in the syscall description language docs.